docs/authorization.md

```diff
@@ -0,0 +1,171 @@
+It holds a list of subjects (users or teams) and a reference to the role being granted. A *RoleBinding* grants permissions within a specific namespace whereas a *GlobalRoleBinding* grants that access global-wide.
+
+A *RoleBinding* may reference any *Role* in the same namespace. Similarly, a *ClusterRoleBinding* can reference any *ClusterRole*.
+
+
+### RoleBinding example
+
+Here is an example of a *RoleBinding* that grants the "dashboard-editor" *Role* to the user "jane" within the "MySuperProject" project. This allows "jane" to edit dashboards in the "MySuperProject" project.
+```yaml
+kind: RoleBinding
+metadata:
+  name: edit-dashboards
+  project: MySuperProject
+spec:
+  role: dashboard-editor
+  subjects:
+    - kind: User
+      name: jane
+```
+
+
+### GlobalRoleBinding example
+
+Here is an example of a *GlobalRoleBinding* that grants the "variable-editor" *GlobalRole* to the user "jane" within all projects. This allows "jane" to edit variables all projects.
+```yaml
+kind: GlobalRoleBinding
+metadata:
+  name: edit-variables
+spec:
+  role: variable-editor
+  subjects:
+    - kind: User
+      name: jane
+```
+
+
+### RoleBinding and GlobalRoleBinding update restriction
+
+After you create a binding, you cannot change the *Role* or *GlobalRole* that it refers to. If you try to change a binding's role, you get a validation error. If you do want to change the role for a binding, you need to remove the binding object and create a replacement.
+
+There are two reasons for this restriction:
+- Making role immutable allows granting someone update permission on an existing binding object, so that they can manage the list of subjects, without being able to change the role that is granted to those subjects. 
+- A binding to a different role is a fundamentally different binding. Requiring a binding to be deleted/recreated in order to change the roleRef ensures the full list of subjects in the binding is intended to be granted the new role (as opposed to enabling or accidentally modifying only the roleRef without verifying all of the existing subjects should be given the new role's permissions).
+
+
+## Referring to resources
+
+In Perses API, resources are identified and accessed using a string, corresponding to the name in the metadata. You can also refer to all resource using the wildcard `*` character.
+Here is an example for granting edit permissions to all resources in all projects:
+```yaml
+kind: GlobalRole
+metadata:
+  name: admin-editor
+spec:
+  permissions:
+    - action: edit
+      scopes: ["*"]
+```
+
+
+## RBAC Synchro
+
+Roles and RoleBindings of an user are stored in the user's JWT.
+If Perses is deployed with multiple instances, Perses RBAC roles and role bindings cache need to be synchronized/replicated between all instances.
+To do that there are multiple mechanisms:
+- cache is refreshed every X minutes
+- cache is refreshed if roles and role bindings retrieve from the user's JWT are different from the cache
```

Comment:
I would say it is refreshed only if the token is saying the user has more right than what is contained in the cache.